src/js/components/inspectors/AutoComplete.tsx

```diff
@@ -104,23 +104,27 @@ var AutoComplete = createReactClass({
 
     str = this.wrapStr(str, SPAN_OPEN, SPAN_CLOSE);
     return str;
-  },
+  };
 
-  htmlToTmpl: function(html) {
+  public htmlToTmpl(html) {
     html = html.split(SPAN_OPEN).join('');
     html = html.split(SPAN_CLOSE).join('');
     return this.htmlDecode(this.wrapStr(html, TMPL_OPEN + DATUM, TMPL_CLOSE));
-  },
+  };
 
-  wrapStr: function(str, pre, post) {
-    var fields = this.props.fields,
-        extraLen = pre.length + post.length,
-        field, position, searched, nextStr;
+  public wrapStr(str, pre, post) {
+    const fields = this.props.fields;
+    const extraLen = pre.length + post.length;
+    let field;
+    let position;
+    let searched;
+    let nextStr;
 
-    for (var i = 0; i < fields.length; i++) {
+    // for (let i = 0; i < fields.length; i++)
+    for (const i of fields.length) {
```

Comment:
could you double check this loop? i think you meant:

`for (const field of fields)` and delete the next line

or just leave it as it was